packages/governance/src/governor/extensions/governor_counting_simple.cairo

```diff
@@ -0,0 +1,151 @@
+// SPDX-License-Identifier: MIT
+// OpenZeppelin Contracts for Cairo v0.19.0
+// (governance/governor/extensions/governor_counting_simple.cairo)
+
+/// # GovernorCountingSimple Component
+///
+/// Extension of GovernorComponent for simple vote counting with three options.
+#[starknet::component]
+pub mod GovernorCountingSimpleComponent {
+    use crate::governor::GovernorComponent::{
+        InternalTrait, ComponentState as GovernorComponentState
+    };
+    use crate::governor::GovernorComponent;
+    use openzeppelin_introspection::src5::SRC5Component;
+    use starknet::ContractAddress;
+    use starknet::storage::{Map, StoragePathEntry, StorageMapReadAccess, StorageMapWriteAccess};
+    use starknet::storage::{StoragePointerReadAccess, StoragePointerWriteAccess};
+
+    type ProposalId = felt252;
```

Comment:
Have you considered moving this typealias to interface and making it public? I thing it could improve functions signatures readability and just make code a bit cleaner throughout the extensions

Reply:
I added the type in here so it is easier to get what is the felt252 representing in the storage member `Governor_proposals_votes: Map<ProposalId, ProposalVote>`. On the other hand, I think it s counterproductive to use it everywhere to represent a proposal id type, since for example a function with a `proposal_id` param is easier to read with `proposal_id: felt252` than with `proposal_id: ProposalId`, since then users need to go (at least one time) to the type to see what is this ProposalId.